pkg/kubelet/pluginmanager/pluginwatcher/README.md

```diff
@@ -30,6 +30,13 @@ setting `maxSurge` to a value larger than zero enables such a seamless upgrade.
 
 **Warning**: Such a seamless upgrade is only supported for DRA at the moment.
 
+## Monitoring plugin connection
+
+If plugin drops GRPC connection with Kubelet, it gets automatically unregistered
```

Comment:
GRPC -> gRPC

Can you add some more meat to this section? How does it work technically? What timeouts are there?

This document is mostly for developers, not users.

Reply:
Added, PTAL.